datahub-web-react/src/app/entity/shared/containers/profile/sidebar/Ownership/AddOwnersModal.tsx

```diff
@@ -0,0 +1,294 @@
+import React, { useEffect, useState } from 'react';
+import { Button, Form, message, Modal, Select, Typography } from 'antd';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+import {
+    CorpUser,
+    EntityType,
+    OwnerEntityType,
+    OwnershipType,
+    SearchResult,
+} from '../../../../../../../types.generated';
+import { useEntityRegistry } from '../../../../../../useEntityRegistry';
+import { CustomAvatar } from '../../../../../../shared/avatar';
+import analytics, { EventType, EntityActionType } from '../../../../../../analytics';
+import { OWNERSHIP_DISPLAY_TYPES } from './ownershipUtils';
+import { useAddOwnersMutation } from '../../../../../../../graphql/mutations.generated';
+import { useGetSearchResultsLazyQuery } from '../../../../../../../graphql/search.generated';
+import OwnerPill from '../../../../../../shared/OwnerPill';
+
+const SearchResultContainer = styled.div`
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 2px;
+`;
+
+const SearchResultContent = styled.div`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+`;
+
+const SelectInput = styled(Select)`
+    > .ant-select-selector {
+        height: 36px;
+    }
+`;
+
+const SearchResultDisplayName = styled.div``;
+
+type Props = {
+    urn: string;
+    type: EntityType;
+    visible: boolean;
+    defaultOwnerType?: OwnershipType;
+    hideOwnerType?: boolean | undefined;
+    onCloseModal: () => void;
+    refetch?: () => Promise<any>;
+};
+
+type SelectedOwner = {
+    label: string;
```

Comment:
It is not clear from this type what the expected values of "label" and "value" are -- is there any way to improve the names of this type?

Reply:
Added and Committed